utils/manifest/index.ts

```diff
@@ -0,0 +1,44 @@
+import { BlobDownloadResponseParsed } from "@azure/storage-blob";
+import { Manifest } from "../interfaces";
+
+export function sanitizeName(manifest: Manifest) {
+  let sanitizedName = manifest.short_name;
+  sanitizedName.replace(/[^a-zA-Z0-9]/g, "_");
+
+  var currentLength;
+  do {
+    currentLength = sanitizedName.length;
+    sanitizedName = sanitizedName.replace(/^[0-9]/, "");
+    sanitizedName = sanitizedName.replace(/^\./, "");
+    sanitizedName = sanitizedName.replace(/\.[0-9]/g, ".");
+    sanitizedName = sanitizedName.replace(/\.\./g, ".");
+    sanitizedName = sanitizedName.replace(/\.$/, "");
+  } while (currentLength > sanitizedName.length);
```

Comment:
Can you explain what these lines are doing? I'm concerned we get in an infinite loop if sanitizedName gets to a point it can't shrink anymore.

Reply:
it's not being used as of yet, it's just a copy of what the sanitize name function is doing in the *-lib atm.

Reply:
I don't like it either, it looks pretty brittle and by removing characters until it starts working doesn't makes sense to me TBH.